src/rustdoc_schema.graphql

```diff
@@ -882,21 +937,24 @@ type AssociatedConstant implements Item {
 
   For example:
   ```rust
-  trait BatchIterator<const MIN: usize> {
-    const SIZE: usize = 16;  // `"16"` is the default
-    const LOG_AS: &'static str = "batch";  // `"\"batch\""` is the default, including escaped quotes
-    const MIN_SIZE: usize = MIN;  // "MIN" is the default, referring to the other constant's name
+  const fn five() -> i32 { 5 };
+  struct Years(i32);
+  trait MyTrait<const MINT: usize> {          // rustdocs default field
+    const MIN     : usize        = 16       ; // `"16"`
+    const MIN_SIZE: usize        = MIN      ; // `"MIN"`, referring to the other constant's name
+    const LOG_AS  : &'static str = "batch"  ; // `"\"batch\""`, including escaped quotes
+    const EXPR2_2 : i32          = 2+2      ; // `"_"`
+    const FN_FIVE : i32          = five()   ; // `"_"`
+    const YEAR    : Years        = Years(42); // `"_"`
   }
   ```
 
-  If the associated constant is on a type's inherent impl, the default is always required to be set.
+  If the associated constant is on a type's inherent impl, `default` is always required to be set.
 
-  If the associated constant is set to be equal to another constant, the default holds the name
-  of that other constant.
+  If the associated constant is set
 
-  If the associated constant is set by evaluating a `const` expression, such as `2 + 2` or
-  a `const fn` call, rustdoc's current behavior is to show a default value of `"_"`
-  instead of evaluating the constant value or including the full expression.
+    - to be equal to another constant, `default` holds the name of that other constant.
+    - by evaluating a `const` expression, such as `2 + 2` or a `const fn` call, `default` is `"_"` instead of evaluating the constant value or including the full expression.
```

Comment:
just for line length
```suggestion
    - by evaluating a `const` expression, such as `2 + 2` or a `const fn` call, 
      `default` is `"_"` instead of evaluating the constant value or including the full expression.
```